example/ExampleApi/Program.cs

```diff
@@ -19,7 +19,7 @@
     builder.Configuration.GetSection("BlobConfiguration").Bind(config);
     config.BlobName = "Folder/settings.json";
 });
-builder.Services.Configure<ExampleOptions>(builder.Configuration.GetSection("ExampleSettings"));
+builder.Services.Configure<ExampleOptions>(builder.Configuration.GetSection("WorkflowDefinitions"));
```

Comment:
I think this can stay the same. The workflowdefintions are specific to nexxbiz not the package.